keyboards/rmi_kb/equator/readme.md

```diff
@@ -0,0 +1,26 @@
+# EQUATOR
+
+![EQUATOR](https://i.imgur.com/0zjj6btl.png)
```

Comment:
```suggestion
![EQUATOR](https://i.imgur.com/0zjj6btlh.png)
```

Reply:
Oh, weird. Not sure what is up with imgur. The original URL is https://i.imgur.com/0zjj6bt.png but I can't seem to get any of the smaller sized images from the URLs (except for `0zjj6bts.png`) - just leads me to a 404.